### ISSUE TITLE
serial capture?

### ISSUE THREAD

**Issue by raszpl** [AkBKukU/CM153-Repro#3]
Great job!
1/ Now that you have a fully working replica, have you tried capturing and decoding serial stream between drive and controller? Looking at https://twitter.com/andyAVAVsystems/status/1192847018483425280/photo/2
there are 3 serial streams? (or one and 2 pins for controlling direction?) + clock + attention (irq like?) and 4 mysterious maybe unused pins. Something like Salea logic clone with 8 bits should be enough to grab it in front of DS26LS31/DS26LS32. Looking at internal pictures of CM100 it has to be some very basic protocol, I wouldnt be surprised if all it does is sending raw CD data straight from EFM decoder, and the rest is handled in the driver.

2/ Could you post the diagram in pdf/png format so people can readily see it when visiting the repo without downloading/configuring kicad? This would also help with any potential referencing it on the forums. Easier to link someone a png than ask them to install kicad.

ps: I posted links to yt and this repo on https://forum.vcfed.org/index.php?threads/philips-cm100-the-first-cd-rom-found.35616 (post waiting for moderation) as that seemed like the place with highest concentration of people talking about CM100 :) You mentioned not being the best with self promotion - I think changing title of YT clip to emphasize this is a replica of a controller for the FIRST computer CDROM drive EVER MADE would help :) Some sources like wikipedia claim Denon & Sony showing first cdrom at 1985 Japan comdex, but the unit in Computer Chronicles clip is a Sony CDU-1 introduced in 1985. CHM https://www.computerhistory.org/storageengine/philips-demonstrates-digital-compact-disc/
"In 1984 DEC introduced the first computer connected CD-ROM player from a major systems manufacturer; the Philips designed RRD-50 (Philips LMSI CM100).". Looks like Wiki needs correction.
btw LGR did a video about another very early drive, Hitachi CDR-1502S from 1986 https://www.youtube.com/watch?v=yBupNdYe08g (error in video claiming its from 1987, his unit was but plenty of references in 1986 press).

ps2 DEC RRD50 Digital Disk Drive Users Guide https://manx-docs.org/collections/antonio/dec/rrd50ug2.pdf

**Comment by AkBKukU** [comment]
1/ It would probably be better to disassemble the driver and work from there to figure out the protocol if someone really wants to. The interface with the 8251 is documented in the datasheets so that wouldn't be hard. That would more likely be complete as well because I wouldn't know how to make the driver send all possible commands to the drive but they would all be in the driver itself.

2/ There is a PDF of the schematic in the folder with the EDA files: https://github.com/AkBKukU/CM153-Repro/blob/main/EDA/ISA-Card/ISA-Card.pdf The gerbers are there as well if someone wants to view the board with a different program as well.

The 1985 drives are prototypes (and absolutely amazing, I want a full height internal CD drive *so bad*) but weren't actually available yet. The RRD-50 was, to the best of my knowledge, the first *released* drive and is the same as the CM100. I intentionally didn't stress the drive too much in the title, or video for that matter, because I don't want it to be confused with the proper video about the CM100 later. If I make the title/thumbnail of this video all about the CM100 and not the card then people might get the later video only about the CM100 confused with this one. And this one seems to be doing fairly well as is so I don't plan on making any changes based on that.

**Comment by raszpl** [comment]
You are absolutely right about completeness of the disassembly approach. There are people who prefer to reason about the code, and those who rather measure first to get a five mile high overview :) A glance at LA capture of the traffic would be fascinating and offer immediate general view of the protocol (speed, size of exchanged packets, general data format) without the need to really dig into the code. If nothing else it would make a nice screenshot :) or even separate video.
EDIT: Finally had a look at the diagram, interesting and potentially fast design (dma). Sadly decompiling the driver wont reveal the speed of Data Clock :( thus the speed of data transfers.
btw ISA-Card.pdf is somehow corrupted in Chrome build-in pdf viewer. Copying part names/netlists copies random garbage. Works fine in Sumatra tho. Is that a kicad bug? I found https://gitlab.com/kicad/code/kicad/-/issues/5593 closed last year.

Im sorry, somehow I missed the pdf :( My brain must of noticed kicad_sch and automatically gave up looking. I would still encourage you to put a direct link to pdf in README.MD above/below parts list. Same goes for link
`[![Cloning a Rare ISA Card to Use a Rare CD Drive](https://img.youtube.com/vi/0W1t2_EJG9w/0.jpg)](https://www.youtube.com/watch?v=0W1t2_EJG9w)`
to your own video. https://stackoverflow.com/questions/11804820/how-can-i-embed-a-youtube-video-on-github-wiki-pages , bonus points for adding red YT PLAY icon in the middle so its obvious to everyone that its a thumbnail for YT clip.
[![Cloning a Rare ISA Card to Use a Rare CD Drive](https://img.youtube.com/vi/0W1t2_EJG9w/0.jpg)](https://www.youtube.com/watch?v=0W1t2_EJG9w)

I filly understand not wanting to clickbait, but stating true facts is a good kind of clickbait :). "Rare ISA Card" "Rare CD Drive" arent exactly brain catchy or informative :( sounds more like "obscure", whereas "first ever" "first commercially available" or even just "first" and an actual model "Philips CM-153" are not only factual, but also immediately tells me what I am clicking at. I would hate to try and find link to this video in 5 years in my browsing history.
At least put "Philips CM-153 LMSI Reproduction Card" in the video description. As of now searching for that phase on google /video will not find _your own video_ because there isnt a single mention of "CM-153" or "LMSI" in title/description :(
